adapter.go

```diff
@@ -99,29 +110,29 @@ type NamedOnceMutex interface {
 	Unlock(key interface{})
 }
 
-// Adapter caches fixed-sized chunks of a KeyReaderAt, and exposes a proxy KeyReaderAt
-// that feeds from its internal cache, only falling back to the provided KeyReaderAt whenever
+// Adapter caches fixed-sized chunks of a KeyStreamerAt, and exposes a proxy KeyStreamerAt
```

Comment:
Adapter exposes a `ReadAt(key string, buf []byte, offset int64)(int,error)` not a KeyStreamerAt

Reply:
Indeed. Should I let `Deprecated` on the interface?